### DISCUSSION TITLE
インライン数式を含む記事を編集すると、勝手に数式が別の表記に置換される

### DISCUSSION THREAD

**Discussion by hmmnrst** [increments/qiita-discussions#952] [要望・アイデア・不具合]
インライン数式で $\theta$ や $\vec{k}$ などを書いておいた下書き記事を編集しようとしたところ、エディタ上ではそれぞれ θ や k→ といったただの文字列に置き換わっていて、しかもそのまま自動保存されて元の記述が失われてしまいました。

**Comment by P-SiZK** [comment]
@hmmnrst 
ご報告ありがとうございます。

MathJax の設定によっては、同様の挙動が報告されております。
添付画像を参考に、MathJax の設定から `Math Settings` -> `Math Renderer` の値をご確認いただけますでしょうか。
もし `Math Renderer` の値が `SVG` になっている場合は、`CHTML` に変更していただき、再度エディタ上での編集をお試しください。

お手数をおかけしますが、よろしくお願いいたします 🙇 

![image](https://github.com/user-attachments/assets/72ba7e37-4409-4d68-b72b-926d886eb4c8)

**Comment by hmmnrst** [comment]
@P-SiZK 
ご回答ありがとうございます。

Math Renderer を CHTML に戻すことで、エディタを開いても数式が維持されることを確認できました。
こちらの設定で利用するようにいたします。
